pyramid_rpc/jsonrpc.py

```diff
@@ -267,7 +267,7 @@ def text(self):
     phash = text
 
     def __call__(self, context, request):
-        return getattr(request, 'rpc_method') == self.method
+        return hasattr(request, 'rpc_method') and getattr(request, 'rpc_method') == self.method
```

Comment:
Ah, I see, add_jsonrpc_method prevents self.method from ever being None. (unless a user manually specifies the predicate, instead of using add_jsonrpc_method, but such a user deserves the trouble.)